.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+version: 2
+updates:
+  - package-ecosystem: github-actions
```

Comment:
What does this do?

Reply:
I'm sorry. I have updated the commit message:

_Dependabot will watch your dependencies and open pull requests when new versions are released. It can update your dependencies automatically and even rebase your pull requests if necessary. Pull requests will be created with monthly frequency._

_doc: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file_

Reply:
In this case, dependencies are GitHub Actions used in workflows.

Reply:
Then LGTM.